patch/0001-v4.19-6-Mellanox-platform-Backport-patches-for-new-M.patch

```diff
@@ -40,39 +41,38 @@ It contains backport patches from v5.* kernels:
 Signed-off-by: Vadim Pasternak <vadimp@mellanox.com>
 Signed-off-by: Stephen Sun <stephens@mellanox.com>
 ---
- drivers/hwmon/mlxreg-fan.c                         |   90 +-
- drivers/hwmon/pmbus/tps53679.c                     |   10 +-
```

Comment:
patches to this file is removed. 

upstream patch applied to 4.9.118

dd19e541758882a2aea3f85412c0ba61b6481056